workflow/workflow_commands.go

```diff
@@ -781,7 +781,7 @@ func convertDescribeWorkflowExecutionResponse(c *cli.Context, resp *workflowserv
 		}
 
 		if pendingActivity.GetHeartbeatDetails() != nil {
-			pendingActivityStr.HeartbeatDetails = stringify.AnyToString(pendingActivity.GetHeartbeatDetails(), true, 0, dataconverter.CustomDataConverter())
+			pendingActivityStr.HeartbeatDetails = stringify.AnyToString(pendingActivity.GetHeartbeatDetails(), true, 0, converter.GetDefaultDataConverter())
```

Comment:
Correctness, `pendingActivity` derives from a response which was passed into this function by `DescribeWorkflow`, which used the standard frontend client factory:

https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/workflow/workflow_commands.go#L687